src/hotspot/share/gc/parallel/psParallelCompact.cpp

```diff
@@ -1913,16 +1913,16 @@ void PSParallelCompact::verify_complete(SpaceId space_id) {
   for (cur_region = beg_region; cur_region < new_top_region; ++cur_region) {
     const RegionData* const c = sd.region(cur_region);
     if (!c->completed()) {
-      log_warning(gc)("region " SIZE_FORMAT " not filled: destination_count=%u",
-                      cur_region, c->destination_count());
+      fatal("region %zu not filled: destination_count=%u",
+             cur_region, c->destination_count());
```

Comment:
I would prefer to use `assert(c->completed(), ...)` in both cases similar to other failures due to verification (like the one above).